app/helpers/admin/corporate_information_pages_helper.rb

```diff
@@ -0,0 +1,48 @@
+module Admin::CorporateInformationPagesHelper
+  include Admin::EditionsHelper
+
+  def index_table_title_row(edition)
+    title = edition.title
+    title += " (#{edition.primary_locale})" if edition.non_english_edition?
+    sanitize(title)
+  end
+
+  def index_table_state_row(edition)
+    state = if edition.force_published?
+              "Force published"
+            elsif edition.unpublishing.present?
+              "Unpublished"
+            else
+              admin_edition_state_text(edition)
+            end
+
+    tags = case state
+           when "Force published"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--yellow")
+           when "Draft"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--blue")
+           when "Submitted"
+             tag.span(state, class: "govuk-tag govuk-tag--s")
+           when "Published"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--green")
+           when "Scheduled"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--turquoise")
+           when "Rejected"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--red")
+           when "Withdrawn", "Unpublished"
+             tag.span(state, class: "govuk-tag govuk-tag--s govuk-tag--grey")
+           end
```

Comment:
I think this can be simplified a lot by doing the following:

```suggestion
    tag_colour_class = case state
           when "Force published" then "govuk-tag--yellow"
           when "Draft" then "govuk-tag--blue"
           when "Published" then "govuk-tag--green"
           when "Scheduled" then "govuk-tag--turquoise"
           when "Rejected" then "govuk-tag--red"
           when "Withdrawn", "Unpublished" then "govuk-tag--grey"
           end
           
    tag.span(state, class: ["govuk-tag govuk-tag--s govuk-tag", tag_colour_class].join(" "))
```

or even

```
    def get_tag_class(state)
        tag_colour_class = case state
               when "Force published" then "govuk-tag--yellow"
               when "Draft" then "govuk-tag--blue"
               when "Published" then "govuk-tag--green"
               when "Scheduled" then "govuk-tag--turquoise"
               when "Rejected" then "govuk-tag--red"
               when "Withdrawn", "Unpublished" then "govuk-tag--grey"
               end
       ["govuk-tag govuk-tag--s govuk-tag", tag_colour_class].join(" ")
    end
    
    tag.span(state, class: get_tag_class(state))
```